### PR TITLE
Mobile-Menu Added

### PR DESCRIPTION
**Pull request by nunegabriel**
In this pull request did the following
- I added a mobile menu
- Setup HTML and CSS linters
- Setup Javascript linters
- Observed best practices for javascript
- added node_modules to my .gitignore

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,27 +12,37 @@
...
+                    <div class="bar-3"></div>
+                </div>
+            </nav>
+            <nav class="curtain curtain-anim" id="curtain">
+                <ul class="tab-items">
+                    <li id="nav1"><a href="#portfolio" class="mobile-links">Portfolio</a></li>
+                    <li id="nav2"><a href="#about" class="mobile-links">About</a></li>
+                    <li id="nav3"><a href="#contact" class="mobile-links">Contact</a></li>
+                </ul>
+            </nav>
```
Comment (ichala) [inline index.html]: - [ ] your mobile menu looks fantastic 💯 , kindly move the links to the top of the menu to match the Figma design: 

Your Design         | Figma Design
:-------------------------:|:-------------------------:
image | image

File: index.html [hunk 2]
```diff
@@ -12,27 +12,37 @@
...
+                    <div class="bar-1"></div>
+                    <div class="bar-2"></div>
+                    <div class="bar-3"></div>
+                </div>
+            </nav>
+            <nav class="curtain curtain-anim" id="curtain">
+                <ul class="tab-items">
+                    <li id="nav1"><a href="#portfolio" class="mobile-links">Portfolio</a></li>
+                    <li id="nav2"><a href="#about" class="mobile-links">About</a></li>
+                    <li id="nav3"><a href="#contact" class="mobile-links">Contact</a></li>
```
Comment (ichala) [inline index.html]: - [ ] Please note that when the user clicks (or taps) any of the mobile menu options, a correct part of the page should be displayed. currently, your links don't work so kindly try to fix that

linkkkk

**Review by ichala — CHANGES_REQUESTED** [review]
Hi @nunegabriel , 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- No linters errors✅ 
- Good Design ✅ 

 ## Required Changes ♻️

- [ ] _**check the comments below**_


## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (victorokoroji) [inline index.html]: - [ ] Good job trying to implement the previous reviewer's suggestion, but it seems you have moved the menu items to the top of the page which is not making your website have a good user experience. Kindly position the menu items to match the Figma design 👍🏽  

Your design 👇🏽 

image
 
Figma Design 👇🏽 

image

**Review by victorokoroji — CHANGES_REQUESTED** [review]
Hi @nunegabriel, 

Great work on making the changes requested by a previous reviewer 👏 🔥 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



## Top highlight 🎯

- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lordkaito — APPROVED** [review]
Hi @nunegabriel, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**